hips/draw/simple.py

```diff
@@ -1,5 +1,24 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HiPS tile drawing -- simple method."""
 
+import healpy as hp
```

Comment:
We said we would use `hips/utils/healpix.py` as wrapper and only import `healpy` there, to make it possible to eventually completely get rid of `healpy`.
I see two options: either expose `hips_nside` as a property (computed from `hips_order`) on the `HipsSurveyProperties` object, or change `compute_healpix_pixel_indices` to take `order` instead of `nside` as parameter (since `order` seems to be the more common quantity available in HipS).